packages/wonder-blocks-link/src/theme/default.ts

```diff
@@ -1,9 +1,12 @@
-import {font} from "@khanacademy/wonder-blocks-tokens";
+import {border} from "@khanacademy/wonder-blocks-tokens";
 
 export default {
     root: {
         font: {
-            weight: font.weight.regular,
+            weight: "inherit",
+        },
+        border: {
+            offset: border.width.medium,
```

Comment:
Would a sizing token be appropriate for the `offset` token? Since the border width token is being used for something other than the border width! The sizing token would be a rem value though 🤔 Curious to hear what you think!